test/controllers/api/v2/content_imports_controller_test.rb

```diff
@@ -74,5 +74,11 @@ def test_version
       post :version, params: { content_view_id: @library_view.id, path: path, metadata: metadata_params}
       assert_response :success
     end
+
+    def test_history_index
+      get :index
+      assert_response :success
+      assert_template 'api/v2/content_view_version_import_histories/index'
+    end
```

Comment:
Can you add a permissions test ? look at -> https://github.com/Katello/katello/blob/master/test/controllers/api/v2/content_exports_controller_test.rb#L23